packages/devtools_app/lib/src/shared/side_panel.dart

```diff
@@ -10,31 +10,42 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_markdown/flutter_markdown.dart';
 import 'package:http/http.dart' as http;
-import 'package:provider/provider.dart';
 
-import '../../../devtools.dart' as devtools;
-import '../../shared/common_widgets.dart';
-import '../../shared/config_specific/launch_url/launch_url.dart';
-import '../../shared/config_specific/logger/logger.dart' as logger;
-import '../../shared/config_specific/server/server.dart' as server;
-import '../../shared/primitives/auto_dispose.dart';
-import '../../shared/theme.dart';
+import '../../devtools.dart' as devtools;
+import 'common_widgets.dart';
+import 'config_specific/launch_url/launch_url.dart';
+import 'config_specific/logger/logger.dart' as logger;
+import 'config_specific/server/server.dart' as server;
+import 'primitives/auto_dispose.dart';
+import 'theme.dart';
 
 const debugTestReleaseNotes = false;
 
-class ReleaseNotesViewer extends StatefulWidget {
-  const ReleaseNotesViewer({
+const releaseNotesTitle = 'What\'s new in DevTools?';
+const releaseNotesTextWhenEmpty = 'Stay tuned for updates.';
+
+const releaseNotesKey = Key('release_notes');
+const diffSnapshotsHelpPanelKey = Key('diff_snapshots_help_panel');
```

Comment:
create an enum to hold the title and key for each view type:
```
enum SidePanelViewType {
  releaseNotes('What\'s new in DevTools?', _releaseNotesKey),
  memoryDiffSnapshotsHelp('Diffing memory snapshots', _memoryDiffSnapshotsHelpKey);

  const SidePanelKey(this.title, this.key);

  final String title;
  final Key key;

  static const _releaseNotesKey = Key('release_notes');
  static const _memoryDiffSnapshotsHelpKey = Key('diff_snapshots_help_panel');
}
```

Then instead of having to pass key and title to SidePanelViewer, you can just pass a single SidePanelViewType value and access title and key from that.

Reply:
You can add `emptyMarkdownFallback` to this too